template/src/pageComponents/mint/ContractDemo.test.tsx

```diff
@@ -0,0 +1,18 @@
+/**
+ * @jest-environment jsdom
+ */
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import OnchainProviders from '../../providers/OnchainProviders';
+import ContractDemo from './ContractDemo';
+
+describe('NotConnected', () => {
```

Comment:
Just a nit. can you wrap this describe with a describe for the `ContractDemo` page.

Reply:
Oops, my bad - made the change 👍